apps/unused_code/unused_code.py

```diff
@@ -53,14 +109,22 @@ def is_ignore_function_list(ignore_prefix_list: list[str], function: ast.Functio
 
 
 def process_file(py_file: str, func_ignore_prefix: list[str], file_ignore_list: list[str]) -> str:
-    if os.path.basename(py_file) in file_ignore_list:
+    if os.path.relpath(py_file) in file_ignore_list:
         LOGGER.debug(f"Skipping file: {py_file}")
         return ""
 
     with open(py_file) as fd:
         tree = ast.parse(source=fd.read())
```

Comment:
`from ast_comments import parse`

```suggestion
        tree = parse(source=fd.read())
```